### PR TITLE
reduce call to get_complete_function_path

### PR DESCRIPTION
**Pull request by xXx-caillou-xXx**
As suggested in: https://github.com/nbs-system/snuffleupagus/issues/193

### CODE REVIEW

File: src/tests/disabled_functions_ret3.phpt [hunk 1]
```diff
@@ -20,5 +20,3 @@ echo("We're at the end of the execution.\n");
 --EXPECTF--
 We're in function `a`.
 [snuffleupagus][0.0.0.0][disabled_function][drop] Aborted execution on return of the function 'Bob::a' in %a/disabled_functions_ret3.php:9, because the function returned '2', which matched a rule.
---XFAIL--
```
Comment (jvoisin) [inline src/tests/disabled_functions_ret3.phpt]: This is unrelated to this PR :/

Comment (xXx-caillou-xXx) [reply src/tests/disabled_functions_ret3.phpt]: Yeah but it fixed it ¯\\\_(ツ)_/¯
